README.md

```diff
@@ -166,13 +166,30 @@ Spin up a new EC2 instance. If using AWS, use the following steps:
 
 ### Deploying Retool on Heroku
 
+#### Automatic deploy
+
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/tryretool/retool-onpremise)
+
+You can deploy to Heroku using the following steps:
+
+1. Click the button above to begin your deployment
+1. Provide a unique App name, e.g. `<your-org>-retool`
+1. Provide required config vars:
+    * `LICENSE_KEY` - your Retool license key
+1. Set any optional config vars:
+    * `USE_GCM_ENCRYPTION` set to `true` for authenticated encryption of
+      secrets; if true, `ENCRYPTION_KEY` must be 24 bytes
+1. Click "Deploy app"
+
+#### Manual deploy
+
 You can manually deploy to Heroku using the following steps:
 
 1. Install the Heroku CLI, and login. Documentation for this can be found here: <https://devcenter.heroku.com/articles/getting-started-with-nodejs#set-up>
 1. Clone this repo `git clone https://github.com/tryretool/retool-onpremise`
 1. Change the working directory to the newly cloned repository `cd ./retool-onpremise`
 1. Create a new Heroku app with the stack set to `container` with `heroku create your-app-name --stack=container`
-1. Add a free database: `heroku addons:create heroku-postgresql:hobby-dev`
+1. Add a database: `heroku addons:create heroku-postgresql:mini`
 1. In the `Settings` page of your Heroku app, add the following environment variables:
    1. `NODE_ENV` - set to `production`
    1. `HEROKU_HOSTED` set to `true`
```

Comment:
as far as I can tell there's no product use of HOST_ENV or HEROKU_HOSTED, so we should be able to remove these variables from the template